spec/Section 7 -- Response.md

```diff
@@ -135,14 +145,103 @@ locations, where each location is a map with the keys `line` and `column`, both
 positive numbers starting from `1` which describe the beginning of an
 associated syntax element.
 
+If an error can be associated to a particular field in the GraphQL result, it
+should contain an entry with the key `path` that details the path of the
```

Comment:
I think the benefits of making error paths a required part of the GraphQL spec outweighs the initial cost of making the change. Without error paths, developers will adopt a variety of ways to get meaningful errors to clients, and the tools that handle those errors won't be interoperable. If we require error paths to be in the spec now, we can easily drop that requirement later, but it will become progressively harder and less useful to change the spec from 'should' to 'must' at some later point.